doc/roadmap.rst

```diff
@@ -56,38 +56,39 @@ The list is numbered not as an indication of the order of priority, but to
 make referring to specific points easier. Please add new entries only at the
 bottom.
 
-#. Everything in Scikit-learn should conform to our API contract
-
-   * `Pipeline <pipeline.Pipeline>` and `FeatureUnion` modify their input
-     parameters in fit. Fixing this requires making sure we have a good
-     grasp of their use cases to make sure all current functionality is
-     maintained. :issue:`8157` :issue:`7382`
 
 #. Improved handling of Pandas DataFrames and SparseDataFrames
```

Comment:
Small nit: you can also remove the "SparseDataFrame" term, as that is being removed in pandas (already the case on master). 
The bullet point of sparse can be kept though, as there are still DataFrames with sparse columns.